app/src/main/java/com/haidoan/android/ceedee/fragmentRentalTabs/TabComplete.kt

```diff
@@ -0,0 +1,124 @@
+package com.haidoan.android.ceedee.fragmentRentalTabs
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
+import android.view.*
+import android.widget.SearchView
+import androidx.core.view.MenuHost
+import androidx.core.view.MenuProvider
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.firebase.firestore.*
+import com.google.firebase.firestore.EventListener
+import com.haidoan.android.ceedee.R
+import com.haidoan.android.ceedee.fragmentRentalTabs.Adapters.RentalAdapter
+import com.haidoan.android.ceedee.fragmentRentalTabs.ViewModels.TabCompleteViewModel
+import java.util.*
+
+
+private lateinit var viewModel: TabCompleteViewModel
+private lateinit var rentalRecyclerView: RecyclerView
+private lateinit var rental_adapter: RentalAdapter
+private lateinit var rentalList: ArrayList<Rental>
+private lateinit var tempRentalList: ArrayList<Rental>
+
+class tabComplete : Fragment() {
```

Comment:
Rename the class ("tabComplete" to "TabComplete"), not just the file name
The easiest way to do this is:
       highlight "tabComplete" -> Right click -> Choose Refactor -> Rename 
This way, Android Studio will rename everything related to this class for you